packages/ocaml-android32.4.02.3/files/config/Makefile.in

```diff
@@ -68,7 +69,6 @@ EXT_LIB=.a
 EXT_DLL=.so
 EXTRALIBS=
 CCOMPTYPE=cc
-TOOLCHAIN=cc
```

Comment:
This value of `TOOLCHAIN` has no relation to the `conf-android:toolchain` variable and instead differentiates GNU compilers on *nix from MinGW and MSVC. Try grepping OCaml makefiles for `TOOLCHAIN`.